linux/ibus-keyman/build.sh

```diff
@@ -33,6 +33,8 @@ else
 fi
 MESON_PATH="../build/$(uname -m)/$MESON_TARGET"
 
+cd "$THIS_SCRIPT_PATH"
+
 builder_describe_outputs \
   configure "${MESON_PATH}/build.ninja" \
   build "${MESON_PATH}/src/ibus-engine-keyman" \
```

Comment:
I notice you have two build outputs? I don't think this is supported

Reply:
At least it seems to work :smile:

Reply:
I think it'll just ignore one of the outputs

Reply:
If I delete either one of them it'll rebuild.

Reply:
I'm not sure how that could work? Reviewing the source of `builder_describe_outputs` shows essentially it results in: `_builder_dep_path[$action$target]="$path"` So I can't see how it would add both items to the array.

Reply:
Ahh, you're right. I have to run `./build.sh test` to test if this works :smile: - and it doesn't. I tried with `./build.sh build`, but that doesn't consider the build outputs specified in `build.sh` and always calls `ninja` which then rebuilds the outputs if necessary.